fastestimator/network.py

```diff
@@ -112,6 +112,7 @@ def get_scheduled_items(self, mode: str) -> List[Any]:
     def load_epoch(self,
                    mode: str,
                    epoch: int,
+                   ds_id: Union[None, str] = None,
```

Comment:
```suggestion
                   ds_id: Optional[str] = None,
```

Reply:
open questions regarding:
1.  whether this should have a default value
2. the meaning of None given that it's also used as the default key. maybe default key should be empty string?